src/renderer/component/common/form-components/form-field.jsx

```diff
@@ -68,6 +71,8 @@ export class FormField extends React.PureComponent<Props> {
         );
       } else if (type === 'textarea') {
         input = <textarea type={type} id={name} {...inputProps} />;
+      } else if (type === 'checkbox' && useToggle) {
+        input = <Toggle id={name} {...inputProps} />;
```

Comment:
There are only 11 checkboxes in the entire app. Is it worth making this all of them?

Also, does Toggle maintain the keyboard usability of native inputs? If we're replacing native form inputs with custom ones, I'd prefer they function the same as native inputs for accessibility and keyboard use. 

In this case, that'd probably mean that you can still tab to them and that space toggles the value.

Reply:
It was just one checkbox on the settings page that was messed up due to the postfix overlflowing onto a second line. Didn't spend much time on it but yes, all of the checkboxes will use this style (which supports nice usability)